src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DocumentMapping/RazorEditHelper.cs

```diff
@@ -45,13 +48,9 @@ internal static async Task<ImmutableArray<RazorTextChange>> MapCSharpEditsAsync(
 
         textChangeBuilder.AddDirectlyMappedEdits(textChanges, codeDocument, cancellationToken);
 
-        var oldUsings = await AddUsingsHelper.FindUsingDirectiveStringsAsync(
-            originalSyntaxTree,
-            cancellationToken).ConfigureAwait(false);
+        var oldUsings = await GetUsingsDirectivesAsync(originalSyntaxTree, cancellationToken).ConfigureAwait(false);
```

Comment:
In the previous PR `AddUsingsHelper.FindUsingDirectiveStringsAsync` was modified to benefit this feature. If that method is no longer used by this feature, I think it would be best to revert the change, lest it inadvertently introduced a change in behaviour.

Alternatively, fix that method in this PR and keep using it, if it doesn't cause other issues.